packages/data-models/deployment.model.ts

```diff
@@ -127,8 +127,13 @@ interface IContentsEntry {
   size_kb: number;
   modifiedTime: string;
   md5Checksum: string;
-  /** specific path to file when same as relativePath, e.g. asset overrides */
+  localPath?: string;
+  /** specific path to file when not the same as relativePath, e.g. asset overrides */
   filePath?: string;
+  /** External URL to display remote assets when running in browser */
+  url?: string;
+  /** Path to file in local device filesystem, populated after asset download */
+  cachedFilepath?: string;
```

Comment:
nit(blocking)
Same comment as file-utils, recommend creating interface in remote-assets feature code instead of changing here